tools/tscdocgen/main.go

```diff
@@ -267,6 +267,15 @@ func (e *emitter) augmentNode(node *typeDocNode, parent *typeDocNode) {
 	node.Label = createLabel(node, parent)
 	node.CodeDetails = createCodeDetails(node)
 	node.RepoURL = getRepoURL(e.repoURL, node, parent)
+	// ShortText can contain an example snippet that contain placeholders that look like
+	// HTML tags, but aren't. For example, <namespace>/name where <namespace> is the placeholder
+	// for an actual namespace value.
+	if node.Comment.ShortText != "" &&
+		strings.Contains(node.Comment.ShortText, "<") &&
+		strings.Contains(node.Comment.ShortText, ">") {
+		node.Comment.ShortText = strings.ReplaceAll(node.Comment.ShortText, "<", "&lt;")
```

Comment:
I know we replace `<` and `>` with `&lt;` and `&gt;` respectively elsewhere, but I wonder: should we be using https://golang.org/pkg/html/#EscapeString? Which would cover other characters like `&` as well?

Reply:
I think that'd be OK. This also got me thinking if I even need to check for the presence of `<` and `>`? It seems like we ought to be able to simply call `EscapeString` regardless? I guess I was trying to protect against occurrences of `<` and `>` where they are not used as a tag, but that doesn't seem like it should matter at all? What do you think?